### PR TITLE
Add session config to decent_quantizer + dropout layer support + newer op support interface for zcu104

### PR DESCRIPTION
**Pull request by anilmartha**
Hi Jorn,

I have added following items. Could you please review and merge?

1. Add session config to decent quantizer
2. Dropout layer runtime support in tensorflow
3. Use newer interface for zcu104 model

Thanks
Anil

### CODE REVIEW

File: tests/unit/contrib/test_dpuczdx8g.py [hunk 1]
```diff
@@ -143,6 +144,20 @@ def test_compile_conv2d_pool2d(self):
                 "DPUCZDX8G-som",
             ],
         )
+        xcompiler_conv2d_dropout_pool2d_nhwc_oihw_test(
```
Comment (jtuyls) [inline tests/unit/contrib/test_dpuczdx8g.py:147]: Could you also add the case with dropout after pooling? And put it in a separate function specific for testing dropout?

File: python/pyxir/runtime/tensorflow/ops/tf_l1_basic_nn.py [hunk 2]
```diff
@@ -651,3 +651,37 @@ def forward_exec(self, inputs: List[np.ndarray]) -> np.ndarray:
...
         with tf.compat.v1.Session() as sess:
             return sess.run(self.res, feed_dict={self.inpt: inputs[0]})
+
+
+########
+# Dropout #
+########
+
+@rt_register_xlayer_2_tf('Dropout')
+class DroputLayer(rt_layer.BaseLayer, RtLayerTF):
```
Comment (jtuyls) [inline python/pyxir/runtime/tensorflow/ops/tf_l1_basic_nn.py:661]: Can you also add a test for this here: https://github.com/Xilinx/pyxir/tree/master/tests/unit/runtime/tensorflow_rt/ops?

File: tests/unit/runtime/tensorflow_rt/ops/test_tf_l1_basic_nn.py [hunk 3]
```diff
@@ -276,6 +276,26 @@ def test_relu(self):
...
+                                       [3, -5, 1, 0], [1, 9, 3, 4]],
+                                      dtype=np.float32),
+                             (1, 1, 4, 4))]
+
+        layers = X_2_TF['Dropout'](dropout, input_shapes, {})
+
+        assert len(layers) == 1
+        outpt = layers[0].forward_exec(inputs)
+
+        expected_outpt = np.reshape(np.array([[0, 2, 0, 0], [10, 0, 0, 0],
```
Comment (jtuyls) [inline tests/unit/runtime/tensorflow_rt/ops/test_tf_l1_basic_nn.py:293]: Why are the remaining values scaled?  (1 -> 2, 3 -> 6, 5 -> 10, etc)
Also, is this deterministic? I would manually create an array of nonzero values and then afterwards calculate the number of zero values (should be half of the values).

Comment (anilmartha) [reply tests/unit/runtime/tensorflow_rt/ops/test_tf_l1_basic_nn.py:293]: I have made it deterministic by setting seed value. The remaining elements are scaled up by 1.0 / (1 - rate)

**Pull request merged**